test/blocks/firefly-gallery/mocks/body.html

```diff
@@ -0,0 +1,17 @@
+<!-- test/blocks/firefly-gallery/mocks/body.html -->
```

Comment:
Hey @nkthakur48 
I see some self-explanatory comments in other files too (apart from firefly-gallery.test.js). Could you please confirm if we need these?